### DISCUSSION TITLE
no "USER-DEEPMD" in the latest version(2.0.3).

### DISCUSSION THREAD

**Discussion by yanjx001** [deepmodeling/deepmd-kit#1421] [Q&A]
Dear sir,

I would like to compile&use lammps code with deepmd-kit and cannot find the folder "USER-DEEPMD" after compiling deepmd-kit-2.0.3. This folder can be found for the earlier versions, e.g. 2.0.0. How to fix it?

Best regards
Jiaxu Yan

**Comment by njzjz** [comment]
It's available in the [manual](https://github.com/deepmodeling/deepmd-kit/releases/tag/v2.0.3), pp. 15. Follow the first command after compiling the library.

**Reply by yanjx001** [reply]
If I installed off-line packages (sh deepmd-kit-2.0.3.sh) or via conda, there is no build directory and thus one cannot run 'make lammps' to generate the USER-DEEPMD module. For older versions, it can automatically generate the USER-DEEPMD module under deepmd-kit/share directory...

**Reply by njzjz** [reply]
The off-line package has switched to the plugin mode.

**Comment by xn-huang** [comment]
I have the same problem.
When I install the latest version of deepmd-kit. There is no user-deepmd package installed in lammps.
I try to make lammps from the deepmd_source_dir. But there is no build directory.

But when I install the version v2.0.0. After I run lmp -h. Error occurs as follows.
lmp: relocation error: ./.conda/envs/dpmd_v200/bin/../lib/libtensorflow_cc.so.2: symbol GOMP_parallel, version VERSION not defined in file libiomp5.so with link time reference

If you know how can I fix it? Thanks.
